Mathlib/RingTheory/MvPolynomial/Symmetric.lean

```diff
@@ -309,4 +312,244 @@ theorem psum_isSymmetric (n : ℕ) : IsSymmetric (psum σ R n) := rename_psum _
 
 end PowerSum
 
+section Exponent
+
+open Classical
+open Composition
+open Finset
+
+variable (σ R) [CommSemiring R] [CommSemiring S] [Fintype σ] [Fintype τ]
+
+@[ext]
+structure Exponent (n : ℕ) :=
+  (exp : σ → ℕ)
+  (sum_eq : ∑ i, exp i = n)
```

Comment:
Honestly, I did not expect `Exponent` to survive this PR, I suspected that there were alternative options but I couldn't find one; I will try to use `Sym` instead. As you say, it might still be more practical to keep `Exponent` and show that there is an equivalence, but I'll try to work without it unless it's really beneficial.